calculator.js

```diff
@@ -0,0 +1,9 @@
+// This is a calculator which takes four variables and perform the four basic mathematical operation
+const num1=5;
+const num2=3;
+const num3=10;
+const num4=15;
+console.log(num1+num4);
+console.log(num2*num3);
+console.log(num3/num2);
+console.log(num4-num1);
```

Comment:
Make sure to put a space after your groupings of variables, and spaces around assignment operators

```const num1 = 5;
const num2 = 3;
const num3 = 10;
const num4 = 15;

console.log(num1 + num4);
console.log(num2 * num3);
console.log(num3 / num2);
console.log(num4 - num1);
```